client/src/pages/SignUp.tsx

```diff
@@ -52,13 +53,12 @@ const SignUpPage = () => {
   useEffect(() => {
     // Only redirect if the user existed before creating the user and after creating the user and assigning him the username
     if (user && stage !== 'updating') {
-      navigate('/selection', {replace: true});
+      navigate('/selection', { replace: true });
```

Comment:
This is a property of navigate, not something I control. What this does is that instead of "pushing" the route to the history of the page, meaning that the user can go back, it "replaces" the current one meaning that the user can't go back to this page when he clicks on back